mongodbatlas/fw_provider.go

```diff
@@ -304,19 +304,13 @@ func setDefaultValuesWithValidations(data *tfMongodbAtlasProviderModel, resp *pr
 			"MONGODB_ATLAS_PUBLIC_KEY",
 			"MCLI_PUBLIC_API_KEY",
 		}, "").(string))
-		if data.PublicKey.ValueString() == "" {
-			resp.Diagnostics.AddError(ProviderConfigError, fmt.Sprintf(AttrNotSetError, "public_key"))
-		}
```

Comment:
Should we print this error message if the AWS credentials are not provided? To make clear that one between PAK and AWS credentials are needed

Reply:
agreed, adjusted PR